torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1107,6 +1107,34 @@ at::Tensor XLANativeFunctions::diag(const at::Tensor& self, int64_t diagonal) {
       tensor_methods::diag(bridge::GetXlaTensor(self), diagonal));
 }
 
+at::Tensor XLANativeFunctions::diag_embed(const at::Tensor& self,
```

Comment:
It looks like the TS backend implemented it via passing it to functionalize again, wdyt?: https://github.com/pytorch/pytorch/blob/c29cfe4d02d9d3479961fc1423f87d1fb29aa959/torch/csrc/lazy/ts_backend/ts_native_functions.cpp#L525

Reply:
I gave it a try, seems like the tests are still passing. Good catch, updated.